### PR TITLE
Add yard email to frontend post request

### LINKED ISSUE #74: Update create yard post request to include email
Add host's email to the creat yard params pass the the engine.
Update necessary tests to reflect the change.

### PR DESCRIPTION
**Pull request by jbranham1**
Close #74 

Thank you everyone for the help dubugging this!!

### CODE REVIEW

File: app/services/engine_service.rb [hunk 1]
```diff
@@ -7,10 +7,12 @@ def self.all_purposes
   end
 
   def self.create_yard(yard_params)
+
```
Comment (abreaux26) [inline app/services/engine_service.rb]: @jbranham1 this is super nitpicky 😅
do you mind removing this extra space?

Comment (jbranham1) [reply app/services/engine_service.rb]: Did this before I saw your comment 😃

**Review by abreaux26 — DISMISSED** [review]
💪🏼

**Review by dev-named-jordan — APPROVED** [review]
Looks good!

**Review by abreaux26 — APPROVED** [review]

**Pull request merged**